app/pages/DashboardPage/DashboardPage.tsx

```diff
@@ -47,7 +50,8 @@ const isEndTimeAfterPresent = (event: IEvent) => new Date(event.end).getTime() >
 const sortByEndTime = (a: IEvent, b: IEvent) => (new Date(a.end).getTime() > new Date(b.end).getTime() ? 1 : -1)
 
 const DashboardPage = () => {
-  const [choosenEvent, setChoosenEvent] = useState<IEventWithCourse>()
+  useForceRender({ intervalTime: ONE_MINUTE_IN_MS })
```

Comment:
Nie można było po prostu zarzucić intervala na filtrowanie zbliżających się eventów? Tylko po to przecież jest wymuszanie rerenderowania

Reply:
Hm, no właśnie nie wiedziałem czy tylko wystarczy czas updatować, czy co, więc walnąłem rerender całego komponentu xD

Reply:
No chyba że tak, już niech zostanie...